### DISCUSSION TITLE
Using Storybook with Rush

### DISCUSSION THREAD

**Discussion by vjau** [storybookjs/storybook#25161] [Help]
### Summary

I had problems installing Storybook in my rush monorepo.
I resolved the problem using following command line
`rush-pnpm dlx storybook@latest init --package-manager=pnpm`
For some reason, even when using the `rush-pnpm` command (or raw pnpm), the install script was still using npm to install dependencies. Adding the `--package-manager=pnpm` option solved the problem. 
I hope this will be useful for others since it took me some time to figure it out.

**Comment by jonniebigodes** [comment]
Hi @vjau, thanks for reaching out to us by opening up this discussion and letting us and the community be aware of this small but rather important issue on setting Storybook with the Rush monorepo architecture. Appreciate it 🙏

**Comment by valentinpalkovic** [comment]
I think this issue was resolved in `8.0.0-alpha.2`. Could you try to execute `rush-pnpm dlx storybook@next init` to see whether the correct package manager gets recognized?

**Reply by vjau** [reply]
Sorry, i hadn't seen your message. 
Trying today `rush-pnpm dlx storybook@next init`, i still get errors 

```
Installing dependencies...

npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "workspace:": workspace:*
```

As you may know, `workspace:` is the special format in `package.json` used by rush to point to local packages in the monorepo.

**Reply by vjau** [reply]
Ok, take my last message with a grain of salt, as i discovered that for some reason, when trying to install storybook in a subrepo of a monorepo with an old version of storybook already installed, it doesn't install the latest version but the same version that is already installed in another subrepo of the monorepo. (i hope that what i just wrote is readable)
